.jenkins/pytorch/test.sh

```diff
@@ -519,11 +519,19 @@ fi
 
 if [[ "$BUILD_ENVIRONMENT" == *coverage* ]]; then
   pushd test
-  echo "Generating XML coverage report"
-  time python -mcoverage xml
+  if [[ -f "./coverage.xml" ]]; then
```

Comment:
Oh no, sorry for the confusion. This should be 
```suggestion
  if [[ -f "./.coverage*" ]]; then
```